.github/workflows/call-style-description.yml

```diff
@@ -0,0 +1,10 @@
+# Style DESCRIPTION file using reusable workflow
```

Comment:
This file could be removed, as this just calls the workflow for the ghactions4r pkg itself (I know, meta). I think it should be removed unless styling the description file is NOT incorporated into the doc-and-style-r workflow.